tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -920,3 +921,139 @@ def test_module_stream_update_from_satellite(session, vm_module_streams):
             status='Upgrade Available',
             stream_version=stream_version
         )
+
+
+@skip_if_not_set('clients', 'fake_manifest')
+@tier3
+def test_syspurpose_attributes_empty(session, vm_module_streams):
+    """
+    Test if syspurpose attributes are displayed as empty
+    on a freshly provisioned and registered host.
+
```

Comment:
I'd expect that the attributes will be set to something non-empty during registration with future iterations of the product, so then this test probably won't make sense. For now it's ok, I count on you to keep an eye on it.

Reply:
Yep, need this to keep this in mind for future, when there will be some syspurp attrs set during the install. Either it should be enough to run the attrs clearing test first, or have a defined set of attributes set during the provisioning and use them in the test.